frontend/src/lib/components/Sidebar/SidebarQuestionList.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+    import StatusIcon from '$icons/Sidebar/CompletionStatus.svelte';
+    import type { QuestionMetadata } from '$types/sidebarMetaData';
+
+    export let questionsMetadata: QuestionMetadata[];
+</script>
+
+<div
+    id="account-accordion"
+    class="hs-accordion-content hidden w-full overflow-hidden transition-[height] duration-300"
+>
+    <ul class="pr-px ps-2 pt-2">
+        {#each questionsMetadata as metadata}
+            <li class="pb-px">
+                <a class="sidebar-question-list-item-style" href="/questions/{metadata.id}">
+                    <StatusIcon isCompleted={metadata.isCompleted} />
+                    {metadata.name}
+                </a>
+                <!-- will need to find a bookmark icon -->
+                <!-- <button on:click={() => console.log('A')}>bookmark</button> -->
```

Comment:
will address the bookmark icon in another PR. Just wanted to merge to main for non blocking purposes for other issues.